### ISSUE TITLE
Blazor Workshop haunts edge browser - firefox is ok

### ISSUE THREAD

**Issue by surfmuggle** [dotnet-presentations/blazor-workshop#332]
I was able to get the blazor workshop running. 

But now it won't get away in edge.

Steps to reproduce

1.  create a directory - for example `Bakery2022` (the name does not matter)
2.  change into that directory and run `dotnet new webapp` and `dotnet build` and `dotnet run`
3. expected: a new webapp is shown (see screenshot of firefox)
4. actual: in edge i tries to load the blazor workshop, in firefox it works as expected

Could anyone help me please to ghostbust the blazor pizza workshop from my edge? 
In other words what do i have to do or delete so that edge will show the current run webapp?

![blazor haunts edge](https://user-images.githubusercontent.com/1271555/161232856-64f83e5e-6c61-4405-afc5-816d26ef734a.png)

**Comment by surfmuggle** [comment]
After the deletion of serveral cached objects (Cached images and files and others) under (see [edge settings](https://support.microsoft.com/en-us/microsoft-edge/view-and-delete-browser-history-in-microsoft-edge-00cf7943-a9e1-975a-a33d-ac10ce454ca4) the problem went away.
